.storybook/test-runner.ts

```diff
@@ -1,28 +1,36 @@
 import { injectAxe, checkA11y } from 'axe-playwright';
 import { Page } from 'playwright-core';
 import { runOnly } from './wcagConfig';
+import { TestContext, TestRunnerConfig } from '@storybook/test-runner';
 
 /**
  * See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api-experimental
  * to learn more about the test-runner hooks API.
  */
-const renderFunctions = {
+const renderFunctions: TestRunnerConfig = {
   async preRender(page: Page) {
     await injectAxe(page);
   },
-  async postRender(page: Page, context) {
-    await checkA11y(page, '#root', {
-      axeOptions: {
-        runOnly,
-        rules: {
-          'color-contrast': { enabled: context.name !== 'Loading' }
-        },
-      },
-      detailedReport: true,
-      detailedReportOptions: {
-        html: true,
+  async postRender(page: Page, context: TestContext) {
+    await checkA11y(
+      page,
+      {
+        include: ['#root'],
+        exclude: ['#root .mapboxgl-canvas-container'],
```

Comment:
For my own edification, was this line the only change needed to get the WCAG tests passing? If we're ignoring Mapbox, the passing of the `MAPBOX_API_KEY` shouldn't really matter, right?

Reply:
Both the passing of mapbox api key and the change in test-runner is needed to get WCAG test passing. It couldn't properly run checks on mapbox stories because the mapbox component never gets render due to no api key, and that causes errors within `play` function and closed the browser/session. With `MAPBOX_API_KEY`, the mapbox stories was evaluated correctly, but there are a couple violations within mapbox canvas, which is what the change in test-runner is for.